.dockerignore

```diff
@@ -10,3 +10,7 @@ node_modules
 .vscode
 .idea
 .*.swp
+
+# Docker
+.dockerignore
```

Comment:
Aren't dotfiles ignored by default, or at least not added if you don't explicitly state you want them?